src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -14,72 +14,78 @@
 /**
  * Class that allows you to exercise Heng Li's fermi-lite assembler.
  * This is backed by JNI native code, which could fail to load.
+ * This class is not thread-safe, but it's very light-weight:  just use a separate instance in each thread.
  */
 public final class FermiLiteAssembler implements AutoCloseable {
     private static volatile boolean nativeLibLoaded = false;
-    private final ByteBuffer opts;
+    private ByteBuffer opts;
 
     public FermiLiteAssembler() {
         loadNativeLibrary();
         opts = createDefaultOptions();
         opts.order(ByteOrder.nativeOrder()).position(0).limit(opts.capacity());
     }
 
+    public boolean isOpen() { return opts != null; }
+
     @Override
-    public void close() { destroyByteBuffer(opts); }
+    public void close() { checkOpen(); destroyByteBuffer(opts); opts = null; }
```

Comment:
I would make it safe to call close repeatedly.  `Autocloseable` has a strong recommendation that close be idempotent.